common/Communication/TLMCommUtil.cc

```diff
@@ -48,10 +48,16 @@ void TLMCommUtil::SendMessage(TLMMessage& mess) {
     // NOTE, "MSG_MORE" flag is important for Linux socket performance!
     int sendBytes = send(mess.SocketHandle, (const char*)&(mess.Header) , sizeof(TLMMessageHeader), MSG_MORE);
 
-    if(sendBytes < 0) {
+    int attempts = 1;
+    TLMErrorLog::Info("First attempt");
+    while(attempts < 10 && sendBytes < 0) {
```

Comment:
Why not use the same code below, where the same pattern is used?

Reply:
Good point, I added it.